design-documents/graph-ql/coverage/re-captcha.graphqls

```diff
@@ -0,0 +1,31 @@
+enum ReCaptchaVersion {
+  V3
```

Comment:
Why have an enum with only one value?

Reply:
Attempt to be forward thinking, if at some point we add support for V2 Invisible or if a V4 ever comes out. Not needed at all, but I didn't think a generic looking `recaptcha` field made sense just for V3.

Reply:
If 3rd-party versioning is a concern, I would consider putting a version label in the type name instead of an interior enum, as values that are currently required may not be available in future versions.  But that might be another discussion I'd want someone else's input on as well

Reply:
Sure, how do I go about facilitating that conversation? I don't care too much about this enum, so if there's an edit you can suggest to get this approved, I'm fine proceeding that way.